files_reader/ajax/personal.php

```diff
@@ -12,20 +12,20 @@
 
 namespace OCA\Files_Reader;
 
-\OCP\JSON::callCheck();
-\OCP\JSON::checkLoggedIn();
+\OC_JSON::callCheck();
```

Comment:
FYI: you replaced a deprecated *public API* with a *private API* that is subject to change at any time without warning (can even happen within minor version of the Nextcloud server because these are not supposed to be used in apps).